cms/static/sass/partials/_variables.scss

```diff
@@ -39,6 +39,11 @@ $f-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
 // ====================
 $transparent: rgba(0,0,0,0); // used when color value is needed for UI width/transitions but element is transparent
 
+// +Colors - UXPL new pattern library colors
+// ====================
+$primary-accent: rgb(14, 166, 236);
```

Comment:
@alisan617 @clrux will update the colors as necessary after the pattern library changes are made. I have made the variable name match the style of the other uxpl colors that Chris recently submitted.